From 26bbb267dee9de1e583f679cac8faeee49e58641 Mon Sep 17 00:00:00 2001
From: George Mocanu <george.mocanu@nxp.com>
Date: Fri, 19 Apr 2024 09:54:29 +0300
Subject: [PATCH] fdts: s32g: Use MSCM#1 interrupt for SCP notifications

Use the MSCM#1 IRQ for SCP SCMI notifications to
SRM. The GoldVIP setup is using the MSCM#0 IRQ in
the context of the multi-core setup of the Real-time
Gateway application, without any option to change
it currently.

Signed-off-by: George Mocanu <george.mocanu@nxp.com>
---
 fdts/s32g.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/fdts/s32g.dtsi b/fdts/s32g.dtsi
index beb306ffa..e804ba8d8 100644
--- a/fdts/s32g.dtsi
+++ b/fdts/s32g.dtsi
@@ -130,11 +130,11 @@
 				"scp_rx_mb", "scp_rx_md",
 				"scmi_ospm_notif";
 			/* <handle, cpn, irq> */
-			nxp,scp-irqs = <&mscm0 M7_0_CPN MSCM_C2C_IRQ_0>,
-				<&mscm0 A53_0_CPN MSCM_C2C_IRQ_0>,
-				<&mscm0 M7_0_CPN MSCM_C2C_IRQ_0>,
+			nxp,scp-irqs = <&mscm0 M7_0_CPN MSCM_C2C_IRQ_1>,
+				<&mscm0 A53_0_CPN MSCM_C2C_IRQ_1>,
 				<&mscm0 M7_0_CPN MSCM_C2C_IRQ_1>,
-				<&mscm0 A53_0_CPN MSCM_C2C_IRQ_1>;
+				<&mscm0 M7_0_CPN MSCM_C2C_IRQ_2>,
+				<&mscm0 A53_0_CPN MSCM_C2C_IRQ_2>;
 			nxp,scp-irq-names = "scp_tx", "scp_rx", "scp_psci", "scp_ospm",
 				"scp_ospm_rx";
 			nxp,notif-irq = <GIC_SPI 300 IRQ_TYPE_EDGE_RISING>;
-- 
2.25.1

